pkg/apis/extensions/latest/latest.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package latest
```

Comment:
> Now we know that there are only two places in our code that handle this correctly and we can start fixing them.

You say "two" above. We will have dozens of groups or more, and not all of them will live in the main repo. Do you still think this is a good pattern?

(The only valid numbers in sentences like the one I quoted are "0", "1", and "infinity"-- and we round up :) )

I'm not going to go for any solution that involves a `latest`-per-group.